src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -378,4 +385,114 @@ public InputStream body() throws IOException {
         }
     }
 
+    /**
+     * Fake decorator.
+     * @since 0.16
+     */
+    final class Fake implements RqMultipart {
+
+        /**
+         * Fake multipart request.
+         */
+        private final RqMultipart fake;
+
+        /**
+         * Fake ctor.
+         * @param req Fake request header holder
+         * @param dispositions Fake request body parts
+         * @throws IOException If fails
+         */
+        public Fake(final Request req, final String... dispositions)
+            throws IOException {
+            this.fake = new RqMultipart.Base(
+                new RqFake(
+                    RqMultipart.Fake.convert(req.head()),
+                    this.fakeBody(dispositions)
+                )
+            );
+        }
+        /**
+         * Fake ctor.
+         * @param dispositions Fake request body parts
+         * @throws IOException If fails
+         */
+        public Fake(final String... dispositions) throws IOException {
```

Comment:
brr... I don't understand this ctor. it should do just this: `this(new RqFake(), dispositions)`. why it's doing something else?